webrtc/videoroom/lib/videoroom/pipeline.ex

```diff
@@ -46,6 +46,8 @@ defmodule VideoRoom.Pipeline do
   def handle_init([room_id]) do
     play(self())
 
+    Logger.metadata(room: room_id)
```

Comment:
Use https://hexdocs.pm/membrane_core/Membrane.Pipeline.Action.html#t:log_metadata_t/0 instead. Only that way the metadata will be propagated to children